app/models/resource.rb

```diff
@@ -8,4 +12,39 @@ class Resource < ActiveRecord::Base
   has_many :services
   has_many :ratings
   has_many :change_requests
+
+  algoliasearch per_environment: true do
+    geoloc :address_latitude, :address_longitude
+
+    add_attribute :address do
+      if address.present?
+        {
+          city: address.city,
+          state_province: address.state_province,
+          postal_code: address.postal_code,
+          country: address.country,
+        }
+      else
+        {}
+      end
+    end
+
+    add_attribute :notes do
+     notes.map {|n| n.note } 
```

Comment:
Ruby trick: You can use `&:note` as shorthand for `{|n| n.note}`, so this could just be written as `notes.map(&:note)`. Same with the other attributes below.